nan.h

```diff
@@ -560,6 +560,14 @@ class AsyncResource {
 #endif
 };
 
+inline uv_loop_t* GetCurrentEventLoop() {
+#if NODE_MODULE_VERSION > NODE_9_0_MODULE_VERSION
```

Comment:
This needs to take the minor version into account.  `node::GetCurrentEventLoop()` was introduced in v9.3.0.

Reply:
I've updated the check to allow versions >= 10, 9 with minor versions >= 3, and 8 with minor versions >= 10. Seems node v6.14.2 also supports `GetCurrentEventLoop` as part of the node_api_backport.h header, but it may not be worth the effort to support it given node 6's age.

Reply:
I think this would result in addons built with e.g. 8.10 don't load with Node 8.9 which is usually not expected.

Reply:
@Flarna hmm... perhaps returning to the original `NODE_MODULE_VERSION > NODE_9_0_MODULE_VERSION` is best then? (which would use `node::GetCurrentEventLoop ` only on node 10)

Reply:
>  think this would result in addons built with e.g. 8.10 don't load with Node 8.9 which is usually not expected.

That's not nan's problem to solve.  Nan provides build-time compatibility, not load-time compatibility.  If we can get both that's nice but it's not a requirement.